README.rst

```diff
@@ -37,55 +37,44 @@ Usage Example
 .. code:: python
 
     import time
+    import random
```

Comment:
I think we can get rid of this `import random`

Reply:
Good catch.  Thanks.  Will remove.